tests/manage/monitoring/conftest.py

```diff
@@ -139,14 +199,15 @@ def stop_mgr():
         time.sleep(run_time)
         return oc.get(mgr)
 
-    measured_op = measure_operation(stop_mgr)
+    test_file = os.path.join(measurement_dir, 'stop_mgr.json')
```

Comment:
To improve clarify, could you use full name of the fixture for the filename? Eg. `workload_stop_ceph_mgr.json`.

Reply:
done